Katello/katello
# Fixes #26417 - Fix Sync Plan dates

This requires https://github.com/theforeman/foreman/pull/6683

This change effectively reverts https://github.com/Katello/katello/pull/5893 as it exposed a date via our API that was inconsistent with other date formats in trying to fix a client side problem.

Now, the dates for the sync plans API are consistent with the rest, making things easier should we ever move to ISO8601 across the board.

To test:

- apply above foreman PR and this PR
- sync plan list should show dates properly
- sync plan create should work
- sync plan update should work (specifically the date & time picker)
